tests/unit/test_bridge_unit.py

```
@@ -0,0 +1,185 @@
+        },
+    ):
+        spec = importlib.util.spec_from_file_location("bridge_under_test", BRIDGE_PATH)
+        module = importlib.util.module_from_spec(spec)
+        assert spec is not None and spec.loader is not None
+        spec.loader.exec_module(module)
+    return module
+
+
+class BridgeUnitTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.9)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)